python/cudf/cudf_pandas_tests/test__main__.py

```diff
@@ -0,0 +1,44 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import subprocess
+import tempfile
+
+
+def test_run_cudf_pandas_with_cli():
```

Comment:
This is a helpful test (I don't think we have coverage here yet) but the name is incorrect. This is just testing the regular "file input" form, not the REPL feature I added in this PR.

So let's rename it:
```suggestion
def test_run_cudf_pandas_with_script():
```

To test the REPL, you'd have to figure out how to send input text to a REPL session, like

```bash
$ python -m cudf.pandas
>>> # Commands get typed here
```

I don't know how to test that off the top of my head.

Reply:
> I don't know how to test that off the top of my head.

You can open the subprocess with a bidirectional pipe and send lines to its `stdin`.

Reply:
Ping @bdice for a review